src/Mod/Sketcher/Gui/DrawSketchHandlerLineSet.h

```diff
@@ -767,6 +765,23 @@ class DrawSketchHandlerLineSet: public DrawSketchHandler
         return QStringLiteral("Sketcher_Pointer_Create_Lineset");
     }
 
+    std::list<Gui::InputHint> getToolHints() const override
+    {
+        using Gui::InputHint;
+        using enum Gui::InputHint::UserInput;
+
+        switch (Mode) {
+            case STATUS_SEEK_First:
+                return {InputHint {QObject::tr("%1 pick first point"), {MouseLeft}}};
+            case STATUS_SEEK_Second:
+                return {InputHint {QObject::tr("%1 pick next point"), {MouseLeft}},
+                        InputHint {QObject::tr("%1 finish"), {MouseRight}},
+                        InputHint {QObject::tr("%1 change mode"), {KeyM}}};

```

Comment:
switch mode? We should be consistent. Maybe we should have some factory for common hints?